src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorProjectChangePublisher.cs

```diff
@@ -113,6 +116,34 @@ internal void ProjectSnapshotManager_Changed(object sender, ProjectChangeEventAr
                 return;
             }
 
+            // Prior to doing any sort of project state serialization/publishing we avoid any excess work as long as there aren't any "open" Razor files.
```

Comment:
Does this prevent Find All References results from Razor files if you haven't  opened a Razor file yet?

Reply:
No it does not. Great question though!